blocks/image-list/image-list.css

```diff
@@ -0,0 +1,72 @@
+main .image-list-container > div, main .image-list-xl-container > div,
+main .image-list-l-container > div, main .image-list-m-container > div,
+main .image-list-s-container > div, main .image-list-xs-container > div {
```

Comment:
I don't think you need the t-shirt sizes here
```suggestion
main .image-list-container > div {
```

Reply:
removed the sizes